projects/peardao/index.js

```diff
@@ -8,7 +8,7 @@ const BUSD_TOKEN_CONTRACT = '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56';
 const USDT_TOKEN_CONTRACT = '0x55d398326f99059ff775485246999027b3197955';
 const USDC_TOKEN_CONTRACT = '0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d';
 const PEX_BNB_LP_CONTRACT = '0x5ca96E8bDe0Bc587DaC9e02422Fd205b1102DAa4';
-const PEX_BNB_LP_STAKING_POOL_CONTRACT = '0x8dBEe838605Df41D947a6634a041A02C74C813b0';
+const PEX_BNB_LP_MASTER_CHEF = '0x73feaa1eE314F8c655E354234017bE2193C9E24E';
```

Comment:
Hi, I noticed that this is Pancakeswap masterchef

But in your UI, I see that you are still pointing to the old masterchef, can you update the UI, then I can merge this :)

Reply:
*Make a comment in this PR after the UI change, I can verify and merge

Reply:
@g1nt0ki Thank you for feedback. There may be some misunderstandings of the product UI, the contract ```PEX_BNB_LP_STAKING_POOL_CONTRACT``` is a nested contract, user stake PEX-BNB LP to this contract, it will transfer the LP to MasterChef immediately, so the we use MasterChef address to perform the calculation, but the direct interactive contract is still ```PEX_BNB_LP_STAKING_POOL_CONTRACT``` as the UI pointed.

Reply:
my bad, thanks for the pr.